solr/core/src/java/org/apache/solr/core/MemClassLoader.java

```diff
@@ -53,6 +53,16 @@ public MemClassLoader(List<PluginBag.RuntimeLib> libs, SolrResourceLoader resour
     this.libs = libs;
   }
 
+  @Override
+  public <T> T newInstance(String cname, Class<T> expectedType, String... subpackages) {
+    return null;
```

Comment:
I suggest adding "nocommit" strings to remind us there is work to do

Reply:
Actually , we are getting rid of `MemClassLoader`. It's a part of the `runtimeLib` feature. So, we  don't need to implement any of these